flagscale/backends/Megatron-LM/megatron/core/parallel_state.py

```diff
@@ -2628,6 +2641,9 @@ def destroy_model_parallel():
     global _VIRTUAL_PIPELINE_MODEL_PARALLEL_WORLD_SIZE
     _VIRTUAL_PIPELINE_MODEL_PARALLEL_WORLD_SIZE = None
 
+    global _DUALPIPEV_PIPELINE_MODEL_PARALLEL_WORLD_SIZE
```

Comment:
The contents that FlagScale adds to the Megatron files, We can add a tag like 
```
######### FlagScale Begin ########
 xxxxxx
######### FlagScale End ########
```
or `######### FlagScale Modify ########` for only one line.